content/blogposts/2023/sourcegraph-5.0.md

```diff
@@ -0,0 +1,250 @@
+---
+title: "Sourcegraph 5.0"
+publishDate: 2023-03-23T10:00-07:00
+description: "Announcing Sourcegraph 5.0. The latest release includes 20+ updates and introduces Cody, the AI programming assistant."
+tags: [blog, release]
+slug: "release/5.0"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/5.0/sourcegraph-5-0-hero.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/5.0/sourcegraph-5-0-hero.png
+changelogItems:
+---
+
+We all know we're headed towards a future where there are a lot more people coding and even more code generated, much of it written by AI. With today's tools, this will be a mess. But with the right tools, this can lead to the most sustained period of human advancement.
```

Comment:
```suggestion
We're headed towards a future with a lot more people and AIs writing a lot more code. With today's tools, this will be a mess. But with the right tools, this can lead to the most sustained period of human advancement.
```